projects/plugins/jetpack/tests/e2e/lib/pages/wp-admin/block-editor.js

```diff
@@ -2,16 +2,12 @@
  * Internal dependencies
  */
 import Page from '../page';
-/**
- * WordPress dependencies
- */
-import { getAllBlocks, searchForBlock } from '@wordpress/e2e-test-utils';
-import { waitAndClick, waitForSelector, scrollIntoView } from '../../page-helper';
 import { getTunnelSiteUrl } from '../../utils-helper';
+import { searchForBlock } from '@wordpress/e2e-test-utils';
```

Comment:
We probably would need to migrate away from this package at some point, since it is designed to be used with puppeteer tests. I'll add task into asana